backend/application/syson-application/src/test/java/org/eclipse/syson/application/publication/SySONLibraryPublicationTests.java

```diff
@@ -65,16 +63,10 @@
  * @see SysONLibraryPublicationListener
  * @author flatombe
  */
-@Transactional(propagation = Propagation.NEVER)
+@Transactional
 @SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
-@ExtendWith(OutputCaptureExtension.class)
-@TestInstance(Lifecycle.PER_CLASS)
 public class SySONLibraryPublicationTests extends AbstractIntegrationTests {
```

Comment:
I reworked the entire class to make it fits the way tests are done in Sirius Web. The main issue was that the tests were calling the `SysONLibraryPublicationHandler` instead of the `PublishLibrariesMutationRunner`. This meant that the whole pipeline wasn't properly tested (i.e. the test could not detect if the data fetcher was able to find the handler).
I also uniformized the `@DisplayName` with the other tests we have, and renamed the methods appropriately.